examples/active_replication/flux_buffer.py

```diff
@@ -0,0 +1,54 @@
+
+class Buffer:
+    def __init__(self, n_dest):
+        self.n_dest = n_dest
+        self.queue = list()  # TODO(yika): add upper bound on buffer size
+
+    def put(self, data, sn, dest):
```

Comment:
Based on the implementation in the paper, same data delivered to different dest are considered different data stored in ingress buffer, so that ack from different dest will drop different data in the buffer